/**:
  ros__parameters:
    can_assign_matrix:
      #UNKNOWN, CAR, TRUCK, BUS, BICYCLE, MOTORBIKE, PEDESTRIAN, ANIMAL
      [0,       0,   0,     0,   0,       0,         0,          0,    #UNKNOWN
       1,       1,   1,     1,   0,       0,         0,          0,    #CAR
       1,       1,   1,     1,   0,       0,         0,          0,    #TRUCK
       1,       1,   1,     1,   0,       0,         0,          0,    #BUS
       0,       0,   0,     0,   1,       1,         1,          0,    #BICYCLE
       0,       0,   0,     0,   1,       1,         1,          0,    #MOTORBIKE
       0,       0,   0,     0,   1,       1,         1,          0,    #PEDESTRIAN
       0,       0,   0,     0,   0,       0,         0,          1]    #ANIMAL
    max_dist_matrix:
      #UNKNOWN, CAR, TRUCK, BUS, BICYCLE, MOTORBIKE, PEDESTRIAN, ANIMAL
      [1.0,     1.0, 1.0,   1.0, 1.0,     1.0,       1.0,        1.0,  #UNKNOWN
       4.0,     2.0, 5.0,   5.0, 1.0,     1.0,       1.0,        1.0,  #CAR
       4.0,     2.0, 5.0,   5.0, 1.0,     1.0,       1.0,        1.0,  #TRUCK
       4.0,     2.0, 5.0,   5.0, 1.0,     1.0,       1.0,        1.0,  #BUS
       3.0,     1.0, 1.0,   1.0, 3.0,     3.0,       2.0,        1.0,  #BICYCLE
       3.0,     1.0, 1.0,   1.0, 3.0,     3.0,       2.0,        1.0,  #MOTORBIKE
       2.0,     1.0, 1.0,   1.0, 3.0,     3.0,       2.0,        1.0,  #PEDESTRIAN
       1.0,     1.0, 1.0,   1.0, 1.0,     1.0,       1.0,        1.0]  #ANIMAL
    max_area_matrix:
      #UNKNOWN,  CAR,      TRUCK,    BUS,      BICYCLE,  MOTORBIKE, PEDESTRIAN, ANIMAL
      [10000.00, 10000.00, 10000.00, 10000.00, 10000.00, 10000.00,  10000.00,   10000.00,  #UNKNOWN
          12.10,    12.10,    19.75,    32.40, 10000.00, 10000.00,  10000.00,   10000.00,  #CAR
          19.75,    12.10,    19.75,    32.40, 10000.00, 10000.00,  10000.00,   10000.00,  #TRUCK
          32.40,    12.10,    19.75,    32.40, 10000.00, 10000.00,  10000.00,   10000.00,  #BUS
           2.50, 10000.00, 10000.00, 10000.00,     1.50,     1.50,      1.00,   10000.00,  #BICYCLE
           3.00, 10000.00, 10000.00, 10000.00,     1.50,     1.50,      1.00,   10000.00,  #MOTORBIKE
           2.00, 10000.00, 10000.00, 10000.00,     1.50,     1.50,      1.00,   10000.00,  #PEDESTRIAN
           2.00, 10000.00, 10000.00, 10000.00, 10000.00, 10000.00,  10000.00,   10000.00]  #ANIMAL
    min_area_matrix:
      #UNKNOWN, CAR,   TRUCK, BUS,    BICYCLE, MOTORBIKE, PEDESTRIAN, ANIMAL
      [ 0.000,  0.000, 0.000,  0.000, 0.000,   0.000,     0.000,      0.000,  #UNKNOWN
        3.600,  3.600, 6.000, 10.000, 0.000,   0.000,     0.000,      0.000,  #CAR
        6.000,  3.600, 6.000, 10.000, 0.000,   0.000,     0.000,      0.000,  #TRUCK
       10.000,  3.600, 6.000, 10.000, 0.000,   0.000,     0.000,      0.000,  #BUS
        0.001,  0.000, 0.000,  0.000, 0.100,   0.100,     0.100,      0.000,  #BICYCLE
        0.001,  0.000, 0.000,  0.000, 0.100,   0.100,     0.100,      0.000,  #MOTORBIKE
        0.001,  0.000, 0.000,  0.000, 0.100,   0.100,     0.100,      0.000,  #PEDESTRIAN
        0.500,  0.000, 0.000,  0.000, 0.000,   0.000,     0.000,      0.000]  #ANIMAL
